gatsby/src/pages/schedule.js

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect } from 'react';
```

Comment:
new page to use the Sessionize schedule embed. TODO: make it re-usable next year.

Reply:
New page is working and looks good.